recipes/mamba-press/recipe.yaml

```diff
@@ -0,0 +1,62 @@
+context:
+  name: mamba-press
+  version: "0.0.1"
+  python_min: "3.12"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/mamba-org/mamba-press/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 66cc5bfb3a023911fdbabfe9812f38aadcb3aa872a6ad6ca4b478b9254c309af
+
+build:
+  number: 0
+  noarch: python
+  script: pip install --no-deps --no-build-isolation -vv .
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - uv-build >=0.8.15
+  run:
+    - python >=${{ python_min }}
+    - libmambapy >=2.3.2
+    - py-lief >=0.16.6
+    - wheel
+    - auditwheel
+    - cattrs
+    - pyyaml
+
+tests:
+  - python:
+      python_version: ${{ python_min }}.*
+      imports:
+      - mamba_press
+      - mamba_press.recipe
+      pip_check: true
+  - if: not win
+    then:
+      script:
+        - mamba-press --help
```

Comment:
This should work on windows too right?

I think you need to specify an entry point
https://rattler.build/latest/reference/recipe_file/#python-specific-options